src/icon/chevronIcon.tsx

```diff
@@ -6,9 +6,10 @@ export type ChevronIconProps = Icon &
   Readonly<{
     down?: boolean
     left?: boolean
+    up?: boolean
   }>
 
-export const ChevronIcon = ({ down, left, ...props }: ChevronIconProps) => (
+export const ChevronIcon = ({ down, left, up, ...props }: ChevronIconProps) => (
```

Comment:
nit: We could also introduced a `Direction` enum to handle this. We'll have trouble if we pass multiple props, i.e. `<ChevronIcon down left/>`

Reply:
Done